pkg/apis/operator/v1alpha1/openshiftpipelinesascode_types.go

```diff
@@ -70,6 +70,25 @@ type OpenShiftPipelinesAsCodeList struct {
 
 type PACSettings struct {
 	Settings map[string]string `json:"settings,omitempty"`
+	// AdditionalPACControllers allows to deploy additional PAC controller
+	// +optional
+	AdditionalPACControllers map[string]AdditionalPACControllerConfig `json:"additionalPACControllers,omitempty"`
 	// options holds additions fields and these fields will be updated on the manifests
 	Options AdditionalOptions `json:"options"`
 }
+
+// AdditionalPACControllerConfig contains config for additionalPACControllers
+type AdditionalPACControllerConfig struct {
+	// Enable or disable this additional pipelines as code instance by changing this bool
+	// +optional
+	Enable *bool `json:"enable,omitempty"`
+	// Name of the additional controller configMap
+	// +optional
+	ConfigMapName string `json:"configMapName,omitempty"`
+	// Name of the additional controller Secret
+	// +optional
+	SecretName string `json:"secretName,omitempty"`
+	// Setting will contains the configMap data
+	// +optional
+	Settings map[string]string `json:"settings,omitempty"`
```

Comment:
so we are adding additionalController field inside PAC settings only, if do this it will be infinite loop :D

and the idea is

```
spec:
  additionalPACControllers:
    controllername:
      enable: true
      configMapName:
      secretName:
      settings:
  enable: true
  settings:
  options:
```

they can provide the settings of particular controller, and use options field for other things, if options have the same deployment name etc provided, it will handle in additionalPACController too